app/javascript/angular/code/filtersUtils.js

```diff
@@ -26,25 +26,27 @@ export const oneHourAgo = () =>
     .subtract(1, "hour")
     .format("X");
 
+export const daterangepickerConfig = (timeFrom, timeTo) => ({
+  linkedCalendars: false,
+  timePicker: true,
+  timePicker24Hour: true,
+  startDate: moment
+    .unix(timeFrom)
+    .utc()
+    .format("MM/DD/YY HH:mm"),
+  endDate: moment
+    .unix(timeTo)
+    .utc()
+    .format("MM/DD/YY HH:mm"),
+  locale: {
+    format: "MM/DD/YY HH:mm"
```

Comment:
maybe we can extract `const format = "MM/DD/YY HH:mm"`?

Reply:
Good point. But I'll leave it for later ;)